spec/system/support_interface/see_individual_application_spec.rb

```diff
@@ -31,6 +31,7 @@ def given_i_am_a_support_user
 
   def and_there_are_applications_in_the_system
     @completed_application = create(:completed_application_form, references_count: 2, with_gces: true)
+    SubmitApplication.new(@completed_application).call
```

Comment:
Is this intended to test `chase_referee_at` and `replace_referee_at` ? 👍 for that!

Reply:
It does test them both indirectly. They only get called for submitted applications so the original system specs were not picking the issue up.

We could add some unit tests for these methods.

Reply:
Yes, I agree, but good it is covered now!